driver-examples/src/main/java/com/datastax/driver/examples/basic/Example0003.java

```diff
@@ -0,0 +1,143 @@
+package com.datastax.driver.examples.basic;
+
+import com.datastax.driver.core.*;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * This example shows how to create schema objects,
+ * load data into Cassandra then retrieve and display them.
+ *
```

Comment:
We could have a section here that indicates what each test mutates in the database.